jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/IsomorphismInspector.java

```diff
@@ -30,16 +30,21 @@
 public interface IsomorphismInspector<V, E>
 {
     /**
-     * Get an iterator over all existing (isomorphic) mappings between two graphs.
+     * Get an iterator over all calculated (isomorphic) mappings between two graphs.
      * 
-     * @return an iterator over all existing (isomorphic) mappings between two graphs
+     * @return an iterator over all calculated (isomorphic) mappings between two graphs
+     *
+     * @throws IsomorphismUndecidableException if the isomorphism test was not executed and the inspector cannot decide whether the graphs are isomorphic
```

Comment:
The javadoc about the exception is not in the correct place. Not all algorithms throw this exception. You should override the Javadoc in your inspector.

Use `{@inheritDoc}` and the `@throws` in the implementation.